src/clients_api_coverage.md

```diff
@@ -90,6 +95,27 @@ will be added here for any other such coverage report.
 | GCM               | ✅         |
 | ChaCha20_Poly1305 | ✅         |
 
+### Cipher algorithms
+
+| Algorithm    | Rust client |
+|--------------|-------------|
+| StreamCipher | ❌         |
+| Crt          | ❌         |
+| Cfb          | ❌         |
+| Ofb          | ❌         |
+| Xts          | ❌         |
+| EbcNoPadding | ❌         |
+| CbcNoPadding | ❌         |
+| CbcPkcs7     | ❌         |
+
+### MAC algorithms
+
+| Algorithm | Rust client |
+|-----------|-------------|
+| Hmac      | ❌         |
+| CbcMac    | ❌         |
+| Cmac      | ❌         |
```

Comment:
For things like these algorithms, where the algorithms _are_ supported, but the operations that use them are not, should they be ticks instead?

Reply:
I would say no personally. I think this page could be presented in a better way. The first table kind of assume that the operation is fully supported when we only really support one specific algorithm.
Ideally, we would have one massive matrix of combination of providers X combination of operations X combinations of their input parameters.

Reply:
And now that all single-part operations are defined, we could think of generating such table programatically, nightly, by either defining new operations that would give us the elvel of support for each operation OR by making test for each one of them. That could be a cool internship project 😉